### ISSUE TITLE
Satisfaction Ratings are not embedded in emails

### ISSUE THREAD

**Issue by ValentinK-Valival** [freescout-helpdesk/freescout#2857]
Hello,
We have recently been using FreeScout version 1.8.68, having the Satisfaction Ratings module Version: 1.0.10.
Satisfaction Ratings are set to correct, but are not inserted into sent emails after the second response.

We looked at the code and found out why the system detects every email as the first response and most likely that is why it is not inserted. We can fix the code but we don't want to because we will lose future updates.

Please help to solve the problem

![Screenshot_1105](https://user-images.githubusercontent.com/55182478/227188708-fcd93333-769e-4e35-a3b6-161da86531cc.png)
![Screenshot_1104](https://user-images.githubusercontent.com/55182478/227188712-8302957c-ad5d-43a1-9e8b-7d329a04513d.png)

**Comment by ilyakonovalenko** [comment]
> We looked at the code and found out why the system detects every email as the first response

What are you findings? It should work fine for the second response and so on.

**Comment by ValentinK-Valival** [comment]
> See https://github.com/freescout-helpdesk/freescout/wiki/FreeScout-Modules#5-modifying-modules

I don't know when we will have time to fix the problem. if you fix it until we find time, it would be great.

**Comment by ilyakonovalenko** [comment]
> I don't know when we will have time to fix the problem. if you fix it until we find time, it would be great.

Fix what? Can you please share what cause the issue? You previously said: "We looked at the code and found out why the system detects every email as the first response"

**Comment by ValentinK-Valival** [comment]
Hello,
we figured out what was going on.
We have the following setting, which is preferred by us: Manage -> Settings -> General -> Emails to Customers-> Conversation History -> Do not include previous messages. We changed to Include the last message and Satisfaction started adding up, with the auto-reply counting as well. Actually, the rule in the code doesn't check if this is the first reply actually and next, but looks at what is sent in the letter itself.
![Screenshot_1109](https://user-images.githubusercontent.com/55182478/227607099-6f0ca30e-efb8-442a-b9c9-5ee199e51997.png)
